web/src/i18n/locales/en.json

```diff
@@ -114,22 +114,22 @@
     "goals": {
       "title": "Needs and Objectives",
       "subheader":
-        "List the goals and objectives you're hoping to accomplish as part of this activity.",
+        "List the goals and objectives you intend to accomplish as part of this activity.",
       "helpText":
-        "Please include a brief description of each goal, and how they relate to the vision identified in the Program Summary section\n\n Within the Objectives section describe how the state will measure progress toward the goal identified.",
+        "Please include a brief description of each goal relates to the vision identified in the program summary section.\n\n Within the objectives section, describe how the state will measure progress toward the goal identified.",
       "reminder": "DON'T FORGET: Each goal should have a matching objective.",
       "goalHeader": "Goal #{{number}}",
       "objectiveHeader": "Objective (How the you'll achieve the goal)",
       "helpText":
```

Comment:
propose this as "objectiveheaderhelptext":

Reply:
Fixed